templates/availability/_pagination.html.twig

```diff
@@ -0,0 +1,17 @@
+{% set nowIsCurrent = (form.vars.value.availabilityDomains[0].date|date('o-\\WW')) == ('now'|date('o-\\WW')) %}
+<nav>
+    <ul class="pagination">
+        <li class="page-item{% if nowIsCurrent %} active{% endif %}">
+            <a class="page-link" href="{{ path('user_availability', { week: 'now'|date('o-\\WW') }) }}">
+                <span aria-hidden="true">&laquo;</span>
+                {{ 'monday' | date('d/m/Y') }} au {{ 'sunday' | date('d/m/Y') }}
```

Comment:
maybe you could use a more sexy date format, like `|format_date(pattern="eeee dd MMMM")`

Reply:
Or you can even display "Semaine courante" ?